golang/lib/networks/network_context.go

```diff
@@ -409,3 +435,35 @@ func (networkCtx *NetworkContext) ExecuteBulkCommands(bulkCommandsJson string) e
 	}
 	return nil
 }
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-client/lib-documentation
+func (networkCtx *NetworkContext) GetServices() ([]services.ServiceID, error){
+	response, err := networkCtx.client.GetServices(context.Background(), &emptypb.Empty{})
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "An error occurred getting services' id list")
```

Comment:
Using the same English rules from above, I'd phrase this as, "An error occurred getting the service IDs set"